Error when building or testing the docs

From terminal, I get this error:
```
  ~/gi/qutip-qip   master ❯ cd docs                              mitiqdev
  ~/gi/qutip-qip/docs   master ❯ make doctest                    mitiqdev
Running Sphinx v3.5.4

Extension error:
Could not import extension numpydoc (exception: No module named 'numpydoc')
make: *** [doctest] Error 2
  ~/gi/qutip-qip/docs   master ❯ pip install sphinx numpydoc sphinx_rtd_theme doctest

Requirement already satisfied: sphinx in /opt/miniconda3/envs/mitiqdev/lib/python3.8/site-packages (3.5.4)
Collecting numpydoc
  Using cached numpydoc-1.1.0-py3-none-any.whl (47 kB)
Collecting sphinx_rtd_theme
  Downloading sphinx_rtd_theme-0.5.2-py2.py3-none-any.whl (9.1 MB)
     |████████████████████████████████| 9.1 MB 7.5 MB/s
ERROR: Could not find a version that satisfies the requirement doctest (from versions: none)
ERROR: No matching distribution found for doctest
  ~/gi/qutip-qip/docs   master ❯ make html                       mitiqdev
Running Sphinx v3.5.4

Extension error:
Could not import extension numpydoc (exception: No module named 'numpydoc')
make: *** [html] Error 2
```
Is the following wrong?
```
pip install sphinx numpydoc sphinx_rtd_theme doctest
```

--- a/README.md
+++ b/README.md
@@ -18,26 +18,30 @@ Installation
 ------------
 To install the package, download to source code and run
 ```
-    pip install .
+pip install .
 ```
 under the directory containing the ``setup.py`` file.
 
 If you want to edit the source code, use instead
 ```
-    pip install -e .
+pip install -e .
 ```
 
-To build and test the documentation, the following packages are required:
+To build and test the documentation, additional packages need to be installed:
 
 ```
-    sphinx numpydoc sphinx_rtd_theme doctest
+pip install matplotlib sphinx numpydoc sphinx_rtd_theme
 ```
 
 Under the docs directory, use
 ```
-    make html
+make html
 ```
-to build the documentation.
+to build the documentation, or
+```
+make doctest
+```
+to test the code in the documentation.
 
 Documentation
 -------------

--- a/docs/source/installation.rst
+++ b/docs/source/installation.rst
@@ -28,9 +28,9 @@ In addition
 
 .. code-block:: bash
 
-    sphinx sphinx_rtd_theme doctest
+    sphinx numpydoc sphinx_rtd_theme
 
-are used to build the documentation.
+are used to build and test the documentation.
 
 A few other packages such as LaTeX is used for circuit plotting, please refer to the main documentation section for detailed instruction.
 

--- a/src/qutip_qip/device/cavityqed.py
+++ b/src/qutip_qip/device/cavityqed.py
@@ -91,7 +91,7 @@ class DispersiveCavityQED(ModelProcessor):
         - ``deltamax``: the pulse strength of sigma-x control, default ``1.0``
         - ``epsmax``: the pulse strength of sigma-z control, default ``9.5``
         - ``eps``: the bare transition frequency for each of the qubits,
-          default ``10``
+          default ``9.5``
         - ``delta``: the coupling between qubit states, default ``0.0``
         - ``g``: the coupling strength between the resonator and the qubit,
           default ``1.0``

--- a/src/qutip_qip/device/circuitqed.py
+++ b/src/qutip_qip/device/circuitqed.py
@@ -40,12 +40,14 @@ class SCQubits(ModelProcessor):
           for each pair of superconducting qubits,
           e.g. ``[5.15, 5.09, 5.15, ...]``
         - ``wr``: resonator bare frequency, default ``[5.96]*num_qubits``
+        - ``g``: The coupling strength between the resonator and the qubits,
+          default ``[0.1]*(num_qubits - 1)``
         - ``alpha``: anharmonicity for each superconducting qubit,
           default ``[-0.3]*num_qubits``
         - ``omega_single``: control strength for single-qubit gate,
-          default ``[-0.01]*num_qubits``
+          default ``[0.01]*num_qubits``
         - ``omega_cr``: control strength for cross resonance gate,
-          default ``[-0.01]*num_qubits``
+          default ``[0.01]*num_qubits``
 
     Attributes
     ----------

